cli/lib/kontena/cli/nodes/update_command.rb

```diff
@@ -11,13 +11,15 @@ class UpdateCommand < Kontena::Command
 
     option ["-l", "--label"], "LABEL", "Node label", multivalued: true
     option "--clear-labels", :flag, "Clear node labels"
+    option "--availability", "AVAILABILITY", "Node scheduling availability"
```

Comment:
This should have some convenient reference for the available options:

`option "--availability", "active|drain", "Node scheduling availability"`